targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP-K1/system_clock.c

```diff
@@ -0,0 +1,275 @@
+/* mbed Microcontroller Library
+* Copyright (c) 2006-2019 ARM Limited
+*
```

Comment:
Thanks for the update, one remaining thing is to add to new files `SPDX-License-Identifier: Apache-2.0` or similar. In this file it would be Apache-2.0 (some other files added here are BSD-3-Clause)

Reply:
Does this file need anymore changes?

Reply:
The three other new files are from STMicroelectronics and hence, leaving the license as is. Only one of those files have been changed (PinNames.h). Please confirm.

Reply:
https://github.com/malavikasajikumar/mbed-os/commit/27abeb993aa332c6e499ec97992ed27eee7bfedc